Documentation/policy/language.rst

```diff
@@ -521,6 +521,11 @@ DNS responses seen by Cilium on the node. Multiple selectors may be included in
 a single egress rule. See :ref:`DNS Obtaining Data` for information on
 collecting this IP data.
 
+.. note:: The DNS Proxy is provided in each Cilium agent.
+   As a result, DNS requests targeted by policies depend on the availability
+   of the Cilium agent pod.
+   This includes DNS visibility policies and pod annotations for policy.
```

Comment:
Sorry, I didn't proof read my suggestion before I sent it. This is what I meant:
```suggestion
   This includes DNS visibility policies and pod annotations for visibility.
```

Reply:
Yes, I fixed it already :-)